moncli/api_v2/handlers.py

```diff
@@ -2263,19 +2262,50 @@ def add_users_to_workspace(workspace_id: str,user_ids: list(), kind: WorkspaceSu
                 utc_hours_diff : `int`
                     The user's UTC hours difference.
             
-            
             Optional Arguments
 
                 api_key: `str`
                     The monday.com v2 API user key.
                 kwargs: dict
                     Additional Arguments
-
         """
 
-
         kwargs["workspace_id"]= gql.IntValue(workspace_id)      
         kwargs['user_ids'] = gql.ListValue([int(id) for id in user_ids])
         kwargs["kind"]= gql.EnumValue(kind)
         
-        return execute_query(api_key=kwargs.pop('api_key', None), query_name=ADD_USERS_TO_WORKSPACE, operation_type=gql.OperationType.MUTATION, fields=args, arguments=kwargs)
\ No newline at end of file
+        return execute_query(api_key=kwargs.pop('api_key', None), query_name=ADD_USERS_TO_WORKSPACE, operation_type=gql.OperationType.MUTATION, fields=args, arguments=kwargs)
+
+
+def delete_users_from_workspace(workspace_id: str, user_ids: list(), *args: tuple, **kwargs: dict ):
+    """Allows you to delete users to a workspace.
+
+            Parameters 
+
+                workspace_id: int
+                        The workspace's unique identifier
+                user_ids: `list[str]`
+                        User IDs to subscribe to the workspace
+                *args: tuple
+                    The collection of workspace return fields.
+            
+            Return Fields
+                
+                id: Int	
+                    The workspace identifier;
+                user_ids: List(Str)
+                    List of user_ids
+
+                
+            Optional Arguments
+
+                api_key: `str`
+                    The monday.com v2 API user key.
+                kwargs: dict
+                    Additional Arguments
+                
+    """
+    kwargs["workspace_id"]= gql.IntValue(workspace_id)
+    kwargs['user_ids'] = gql.ListValue([gql.IntValue(id) for id in user_ids])
```

Comment:
I provided improper guidance on this one as well.  The following is the desired operation:

`gql.ListValue([int(id) for id in user_ids])`